app/Http/Controllers/Api/V1/StandAloneActivity.php

```diff
@@ -0,0 +1,527 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use App\Events\ActivityUpdatedEvent;
+use App\Events\PlaylistUpdatedEvent;
+use App\Http\Controllers\Controller;
+use App\Http\Requests\V1\ActivityCreateRequest;
+use App\Http\Requests\V1\ActivityEditRequest;
+use App\Http\Requests\V1\StandAloneActivityCreateRequest;
+use App\Http\Resources\V1\ActivityResource;
+use App\Http\Resources\V1\ActivityDetailResource;
+use App\Http\Resources\V1\H5pActivityResource;
+use App\Http\Resources\V1\PlaylistResource;
+use App\Http\Resources\V1\StandAloneActivityResource;
+use App\Jobs\CloneActivity;
+use App\Models\Activity;
+use App\Models\Playlist;
+use App\Models\Project;
+use App\Repositories\Activity\ActivityRepositoryInterface;
+use App\Repositories\Playlist\PlaylistRepositoryInterface;
+use Djoudi\LaravelH5p\Events\H5pEvent;
+use Djoudi\LaravelH5p\Exceptions\H5PException;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\App;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Facades\Validator;
+use Illuminate\Support\Arr;
+use H5pCore;
+use App\Models\Organization;
+
+/**
+ * @group 5. Activity
+ *
+ * APIs for stand alone activity management
+ */
+class StandAloneActivity extends Controller
+{
+
+    private $activityRepository;
+
+    /**
+     * StandAloneActivity constructor.
+     *
+     * @param ActivityRepositoryInterface $activityRepository
+     */
+    public function __construct(
+        ActivityRepositoryInterface $activityRepository
+    )
+    {
+        $this->activityRepository = $activityRepository;
+
+        // $this->authorizeResource(Activity::class, 'activity');
```

Comment:
remove this if not needed and also the blank line

Reply:
removed